src/components/SUE/report/SueReportServices.tsx

```diff
@@ -14,7 +15,24 @@ export const SueReportServices = ({
   serviceCode: string;
   setServiceCode: any;
 }) => {
-  const { data } = useQuery([QUERY_TYPES.SUE.SERVICES], () => getQuery(QUERY_TYPES.SUE.SERVICES)());
+  const [loading, setLoading] = useState(true);
+
+  const { data, isLoading, refetch } = useQuery([QUERY_TYPES.SUE.SERVICES], () =>
+    getQuery(QUERY_TYPES.SUE.SERVICES)()
+  );
+
+  useEffect(() => {
+    refresh();
+  }, []);
+
+  const refresh = async () => {
+    await refetch();
+    setLoading(false);
+  };
+
+  if (isLoading || loading) {
+    return <LoadingSpinner loading />;
+  }
 
   return (
     <View style={styles.container}>
```

Comment:
and what about pull to refresh? is this view a child of a scroll view? or could this be transformed into a scroll view?